### PR TITLE
Use class name as unique identifier for custom benefits and conditions.

### PR DESCRIPTION
**Pull request by michaelkuty**

### CODE REVIEW

File: src/oscar/apps/offer/custom.py [hunk 1]
```diff
@@ -38,8 +38,10 @@ def create_condition(condition_class, **kwargs):
     """
     Create a custom condition instance
     """
-    return Condition.objects.create(
-        proxy_class=_class_path(condition_class), **kwargs)
+    created, condition = Condition.objects.create(
+        proxy_class=_class_path(condition_class), defaults=kwargs)
```
Comment (mvantellingen) [inline src/oscar/apps/offer/custom.py]: ``Model.objects.create()`` doesn't return a tuple i think?

Comment (sasha0) [inline src/oscar/apps/offer/custom.py]: No it doesn't. Maybe it was meant to use `Condition.objects.get_or_create()`.

**Comment by michaelkuty** [comment]
Yes, you are right

**Comment by michaelkuty** [comment]
You cannot call these metods when app starts because you must check duplicities, from my point of view this is better for custom offers use cases, but agree that change is not important for all users

**Pull request closed without merging**